packages/framework/client-logger/external-telemetry/src/common/telemetry/index.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ContainerTelemetryEventName } from "../../container";
+
+/**
+ * The base interface extended by all external telemetry
+ *
+ * @beta
+ */
+export interface IExternalTelemetry {
+	eventName: ExternalTelemetryEventName;
+}
+
+/**
+ * Aggregate type for all the different types of external telemetry event names.
+ *
+ * @remarks This only looks odd right now because {@link ContainerTelemetryEventName} is the only aggregation at the moment.
```

Comment:
I think this note is more for us than for external consumers.
```suggestion
 * @privateremarks This only looks odd right now because {@link ContainerTelemetryEventName} is the only aggregation at the moment.
```

Reply:
Updated. Nice I didn't know about private remarks